app/Http/Controllers/CommentController.php

```diff
@@ -24,20 +24,41 @@ protected function create(array $data)
     }
 
     /**
-    * Process comment creation
-    */
+     * Process comment creation
+     */
     public function postComment(Request $request)
     {
         $newComment = $this->create($request->all());
-
         return [
-                'message' => 'Comment created Successfully',
-                'status_code' => 200,
-                'commentId' => $newComment->id
+            'message' => 'Comment created Successfully',
+            'status_code' => 200,
+            'commentId' => $newComment->id
             ];
     }
 
     /**
+     * Get 10 comments from current episode
+     */
+    public function fetchComment(Request $request)
```

Comment:
@andela-tolotin Please write tests to assert that  `fetchComment` returns the next comments.